src/pages/dar_application/DataUseAgreements_new.js

```diff
@@ -0,0 +1,81 @@
+
+import { a, div, h2, h3, span, li, ol, p} from 'react-hyperscript-helpers';
+import {isNil} from 'lodash/fp';
+import LibraryCardAgreementLink from '../../assets/Library_Card_Agreement_2021.pdf';
+
+import './dar_application_new.css';
+
+export default function DataUseAgreements(props) {
+
+  const {
+    save,
+    attestAndSend,
+    darCode,
+  } = props;
+
+  return (
+    div({
+      className: 'dar-step-card'
+    }, [
+
+      h2({}, ['Step 4: Attestation and Agreement(s)']),
```

Comment:
The mock has "Data Use Agreements" as the header for this section, which is not the same as the "Data Use Agreement" section on the lefthand nav bar.
<img width="1342" alt="Screen Shot 2022-10-06 at 4 50 37 PM" src="https://user-images.githubusercontent.com/59845076/194416011-b5d24c87-9634-42af-bf80-ed82d6e29dae.png">

Reply:
Another detail I realized when taking this screenshot is that each of the step cards in the mock are separate with white space between each step. Not sure if that change fits into the scope of this PR though cause we don't have that implemented for the previous steps yet either.